### PR TITLE
Requested text changes

### PR DESCRIPTION
**Pull request by msquance-stem**
## Status

* Current Status: Ready for front-end review
* Review App link(s): *populate this with links to the relevant parts of the review app*
* Closes https://github.com/NCCE/teachcomputing.org-issues/issues/2607 https://github.com/NCCE/teachcomputing.org-issues/issues/2608

## Review progress:

- [ ] Browser tested
- [ ] Front-end review completed
- [ ] Tech review completed

## What's changed?

Requested text changes for KS3, and also replacing all accredited by BCS to awarded by BCS

### CODE REVIEW

File: app/views/pages/enrolment/non_gcse.html.erb [hunk 1]
```diff
@@ -12,7 +12,7 @@
             and support you on your journey.
           </p>
 
-          <a href="https://forms.office.com/e/VHwFA9aS3g" class="govuk-button button">Request SME support</a>
+          <a href="https://forms.office.com/e/VSLpEXu3vH" class="govuk-button button">Request SME support</a>
```
Comment (Caleb-T-Owens) [inline app/views/pages/enrolment/non_gcse.html.erb:15]: I've seen this link a few times around the app - could we move it into the external_link_helper?

**Review by Caleb-T-Owens — CHANGES_REQUESTED** [review]
All looks good apart from a link which seems to be appearing in a few places that would be great to move into a helper

File: app/views/pages/enrolment/non_gcse.html.erb (hunk 1 above)
Comment (msquance-stem) [reply app/views/pages/enrolment/non_gcse.html.erb:15]: Good point, will do that now

**Review by Caleb-T-Owens — APPROVED** [review]
Fantastic! Thanks for the updates 👍

**Pull request merged**